src/components/Search.vue

```diff
@@ -1,11 +1,14 @@
 <template lang="pug">
-  b-jumbotron(bg-variant="info" text-variant="white" border-variant="dark")
-    template(slot="header") Testing
-    template(slot="lead")
-      | This is a text that is used to test something. Very interesting! 
-      font-awesome-icon(icon="coffee")
-    hr.my-4
-    p It is very nice
+  .container
```

Comment:
Why do we need a container div here?

Reply:
We need to use a div as the root element and a container is used for proper alignment with bootstrap